src/os/bluestore/BlueFS.cc

```diff
@@ -1306,7 +1346,7 @@ int BlueFS::device_migrate_to_new(
     (!bdev[BDEV_SLOW] ? RENAME_DB2SLOW: REMOVE_DB) :
     0;
   flags |= devs_source.count(BDEV_WAL) ? REMOVE_WAL : 0;
-  int dev_target_new = dev_target;
+  int dev_target_new = dev_target; //FIXME: remove, makes no sense
```

Comment:
So, does dev_target_new remains here as alias or goes away?

Reply:
that was just an uncommitted cleanup TODO. No need for this var since dev_target isn't modified over this function's code.